templates/dashboard/base.html

```diff
@@ -129,7 +129,7 @@
               <ul>
                 <li class="{% block menu_orders_class %}{% endblock %}">
                   <a href="{% url 'dashboard:orders' %}">
-                    {% trans "Orders" %}
+                    {% trans "Orders" context "Dashboard side navigation link" %}
```

Comment:
I suggest we change the context on all these and the matching breadcrumbs to something like "Dashboard section". I think it's more important to have consistent semantics than to be very descriptive about the placement.